src/intel_driver.c

```diff
@@ -73,11 +73,11 @@ bool intel_batchbuffer_configure_watchdog(int fd, unsigned int gem_ctx_id, int f
     memset(threshold, 0, sizeof(threshold));
 
     /* set watchdog threshold to VCS's engine only */
-    if (ring_flag == I915_EXEC_BSD || ring_flag == LOCAL_I915_EXEC_VCS2) {
-        threshold[VIDEO_DECODE_CLASS] = 1200;
+    if (ring_flag == I915_EXEC_BSD || ring_flag == (I915_EXEC_BSD_RING2 | I915_EXEC_BSD)) {
```

Comment:
ring_flag = flag & I915_EXEC_RING_MASK, so it is always false for ring_flag == (I915_EXEC_BSD_RING2 | I915_EXEC_BSD